src/monopoly.js

```diff
@@ -0,0 +1,48 @@
+export class Monopoly {
+  constructor(){
+    this.PropertyList = null;
+    this.ChanceList = null;
+    this.CommChestList = null;
+    this.PlayerList = null;
+  }
+
+  loadPlayer(){}
+
+  runGame(){
+    cleanBoard();
+
+    findFirstPlayer();
+
+    while(PlayerList.length > 1){
+      runTurn(PlayerList.current);
+      PlayerList.Current = PlayerList.Current.Next;//or next index or w.e. I use
+    }
+
+    //add wincount;
+  }
+
+  runTurn(){
+    
+  }
+
+  cleanBoard(){
+    PropertyList = new PropertyList;
+    ChanceList = new ChanceList;
+    CommChestList = new CommChestList;
+
+    resetPlayers();
+  }
+
+  findFirstPlayer(){}
+
+  resetPlayers(){}
+
+  static rollDice(){
+    var dice = new Array(2);
```

Comment:
Idiomatic JS dictates that we would probably use something like map for this.  As you use map more and more you'll become comfortable with it:

```
return (new Array(2)).map(() => {
    return Math.floor(Math.random() * 6) + 1;
});
```